cssom/Overview.bs

```diff
@@ -1457,10 +1462,35 @@ To <dfn>serialize a CSS rule</dfn>, perform one of the following in accordance w
 
 
  <dt>{{CSSMediaRule}}
- <dd>Issue: ...
+ <dd>
+  The result of concatenating the following:
+  <ol>
+   <li>The string "<code>@media</code>" followed by a single SPACE (U+0020).</li>
+   <li>The result of performing <a>serialize a media query list<a> on rule's media query list.</li>
+   <li>A single SPACE (U+0020), followed by the string "{", i.e., LEFT CURLY BRACKET (U+007B), followed by a single SPACE (U+0020).</li>
+   <li>The result of performing <a>serialize a CSS rule</a> on the each rule, separed by a single SPACE (U+0020), in rule's CSSRules list.</li>
+   <li>A single SPACE (U+0020), followed by the string "}", i.e., RIGHT CURLY BRACKET (U+007D)</li>
+  </ol>
 
  <dt>{{CSSFontFaceRule}}
- <dd>Issue: ...
+ <dd>
+  The result of concatenating the following:
+  <ol>
+   <li>The string "<code>@font-face {</code>" followed by a single SPACE (U+0020).</li>
+   <li>The string "<code>font-family:</code>" followed by a single SPACE (U+0020).</li>
+   <li>The result of performing <a>serialize a URL</a> on the rule’s font family name.</li>
+   <li>The string "<code>;</code>", i.e., SEMICOLON (U+003B).
+   <li>
+    If the rule's associated source list is not empty, follow these substeps:
+    <ol>
+     <li>A single SPACE (U+0020), followed by the string "<code>src:</code>", followed by a single SPACE (U+0020).</li>
+     <li>The result of invoking <a>serialize a comma-separated list</a> on performing <a>serialize a URL</a> or <a>serialize a LOCAL<a> for each source on the source list.</li>
+     <li>The string "<code>;</code>", i.e., SEMICOLON (U+003B).</li>
+    </ol>
+   </li>
+   <li>A single SPACE (U+0020), followed by the string "}", i.e., RIGHT CURLY BRACKET (U+007D).</li>
```

Comment:
They weren't implemented in servo yet(where I implement this serialization steps) and I forgot these. I added these too, thanks.